tst/testinstall/trans.tst

```diff
@@ -2295,7 +2295,9 @@ gap> (f * (1, 2)) * (1, 2, 3);
 Transformation( [ 3, 2, 1 ] )
 gap> (1, 2, 3) * (f * (1, 2));
 Transformation( [ 1, 3, 2 ] )
-gap> p := AS_PERM_TRANS(f) * (1,2);;
+
+# Large values force this to be a Perm4Rep
+gap> p := AS_PERM_TRANS(f) * (1,2)(2^16,2^16+1) * (2^16,2^16+1);;
```

Comment:
Why is this change necessary?

Also, I don't really understand what the comment is trying to tell me. Is the *goal* here to force a Perm4Rep, to ensure some tests later on are meaningful? Then perhaps the comment should state it that way, and also give the reason?